app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -906,50 +984,48 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
      */
     private fun showOrbotInstallDialog() {
         val builder = AlertDialog.Builder(requireContext())
-        //set title for alert dialog
         builder.setTitle(R.string.orbot_install_dialog_title)
-        //set message for alert dialog
         builder.setMessage(R.string.orbot_install_dialog_message)
 
         //Play store install intent
-        if(BuildConfig.FLAVOR == Constants.FLAVOR_PLAY){
-            builder.setPositiveButton(getString(R.string.orbot_install_dialog_positive)){ _, _ ->
+        if (BuildConfig.FLAVOR == Constants.FLAVOR_PLAY) {
+            builder.setPositiveButton(getString(R.string.orbot_install_dialog_positive)) { _, _ ->
                 val intent = get<OrbotHelper>().getIntentForDownload(requireContext(), DOWNLOAD_SOURCE_PLAY_STORE)
-                if(intent != null){
+                if (intent != null) {
                     try {
                         startActivity(intent)
-                    }catch (e: ActivityNotFoundException){
-                        Utilities.showToastInMidLayout(requireContext(), getString(R.string.orbot_install_activity_error), Toast.LENGTH_SHORT)
+                    } catch (e: ActivityNotFoundException) {
+                        Utilities.showToastUiCentered(requireContext(), getString(R.string.orbot_install_activity_error), Toast.LENGTH_SHORT)
                     }
-                }else{
-                    Utilities.showToastInMidLayout(requireContext(), getString(R.string.orbot_install_activity_error), Toast.LENGTH_SHORT)
+                } else {
+                    Utilities.showToastUiCentered(requireContext(), getString(R.string.orbot_install_activity_error), Toast.LENGTH_SHORT)
```

Comment:
repeat copy-paste code. move it to its own method.

Also, simply the `if` blocks:

```
fun startOrbotInstallActivity(intent) {
  if (intent == null) {
    Utilities.showToastUiCentered(context, ...)
    return
  }

  try {
    startActivity(intent)
  } catch (e: ActivityNotFoundException) {
    Utilities.showToastUiCentered(context, ...)
  }
  
}
```

Reply:
Created method `startOrbotInstallActivity`. Modified the `if` block as well.